### PR TITLE
Mark Knative serving service and kfservice clear in log

### PR DESCRIPTION
**Pull request by hougangliu**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kfserving/29)

### CODE REVIEW

**Comment by yuzisun** [comment]
@hougangliu Thanks for fixing my logs! I had hard time to name these things.

File: pkg/controller/kfservice/kfservice_controller.go [hunk 1]
```diff
@@ -124,7 +124,7 @@ func (r *ReconcileService) Reconcile(request reconcile.Request) (reconcile.Resul
 
 	desiredService, err := resources.CreateKnativeService(kfsvc)
 	if err != nil {
-		log.Error(err, "Failed to create desired ksvc", "name", desiredService.Name)
+		log.Error(err, "Failed to create desired Knative Serving Service", "name", kfsvc.Name)
```
Comment (ellis-bigelow) [inline pkg/controller/kfservice/kfservice_controller.go]: If we're going to change this, we at least need to be consistent. KFService?

File: pkg/reconciler/ksvc/resources/knative_service.go [hunk 2]
```diff
@@ -37,8 +37,7 @@ func CreateKnativeService(kfsvc *v1alpha1.KFService) (*knservingv1alpha1.Service
 	var revisions []string
 	container := &v1.Container{}
 	routingPercent := 0
-	if kfsvc.Spec.Canary == nil ||
-		(kfsvc.Spec.Canary.TrafficPercent == 0 && kfsvc.Spec.Canary != nil) {
+	if kfsvc.Spec.Canary == nil || kfsvc.Spec.Canary.TrafficPercent == 0 {
```
Comment (ellis-bigelow) [inline pkg/reconciler/ksvc/resources/knative_service.go]: I'm gonna let @yuzisun take this one ;)

File: pkg/controller/kfservice/kfservice_controller.go (hunk 1 above)
Comment (ellis-bigelow) [reply pkg/controller/kfservice/kfservice_controller.go]: I just re-read this XD. Nvm.

Comment (yuzisun) [reply pkg/controller/kfservice/kfservice_controller.go]: I agree, as long as it is consistent whether it is full name or short name, ksvc is a short name for Knative Service.

File: pkg/reconciler/ksvc/resources/knative_service.go (hunk 2 above)
Comment (yuzisun) [reply pkg/reconciler/ksvc/resources/knative_service.go]: fine with me :).

**Comment by ellis-bigelow** [comment]
Thanks @hougangliu! Just fix that name and I'll approve :)

File: pkg/controller/kfservice/kfservice_controller.go (hunk 1 above)
Comment (ellis-bigelow) [inline pkg/controller/kfservice/kfservice_controller.go]:
```suggestion
		log.Error(err, "Failed to create desired Knative Serving Service", "name", desiredService.Name)
```

Comment (hougangliu) [reply pkg/controller/kfservice/kfservice_controller.go]: when `err != nil`,  maybe we have no idea if desiredService is nil or not (although current resources.CreateKnativeService logic returing desiredService is not nil, even err will never be nil), so I think use `kfsvc.Name` instead. Make sense?

Comment (yuzisun) [reply pkg/controller/kfservice/kfservice_controller.go]: yes, you are right.

File: pkg/controller/kfservice/kfservice_controller.go [hunk 3]
```diff
@@ -166,9 +165,9 @@ func (r *ReconcileService) updateStatus(before *kfservingv1alpha1.KFService, ksv
 		// to status with this stale state.
 
 	} else if err := r.Update(context.TODO(), after); err != nil {
-		log.Error(err, "Failed to update kfsvc status")
+		log.Error(err, "Failed to update KFService status")
```
Comment (yuzisun) [inline pkg/controller/kfservice/kfservice_controller.go]: to be consistent with above to use full name, can we say `kfserving service` here?

**Pull request merged**